pkgs/servers/nominatim/default.nix

```diff
@@ -0,0 +1,66 @@
+{
+  stdenv, lib, fetchgit, fetchurl,
+  clang-tools, cmake, bzip2, zlib, expat, boost, git, pandoc, gzip,
+  postgresql_12,
+  python3, python3Packages, php
+}:
+
+let
+  countryGrid = fetchurl {
+    # Nominatim docs mention this URL but it's not a very good one for pinning
+    # url = "https://www.nominatim.org/data/country_grid.sql.gz";
+    # sha256 = "sha256-/mY5Oq9WF0klXOv0xh0TqEJeMmuM5QQJ2IxANRZd4Ek=";
+    url = "https://gist.github.com/mausch/913f456c672d2486e7a53dbba5ea0a47/raw/35d87f84eb4c9477ac0498879e04ca58492e1446/country_grid.sql.gz";
+    sha256 = "sha256-yvm064uehuYxBvmRKViXdDG7aWW5u3ul5hBr4WEpBIE=";
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "nominatim";
+  version = "4.0.1";
+
+  src = fetchgit {
+    url = "https://github.com/osm-search/Nominatim.git";
+    rev = "v${version}";
+    sha256 = "sha256-sKI/KBKveb5kAWJ7y1xw+ZF1thynr402rJhVjkIdFMo=";
+  };
+
+  buildInputs = [
+    cmake
```

Comment:
this goes into nativeBuildInputs with all other binaries that are executed at build time.

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/165183/commits/bc4bf3e4b6d33a1997f44b6a4f495717256eec6d